src/views/terms-and-conditions.html

```diff
@@ -0,0 +1,129 @@
+{% extends 'default.html' %}
+{% set title = 'Terms and Conditions' %}
+{% set footer = true %}
+{% set homeLink = true %}
+{% block pageHeading %}Terms and Conditions{% endblock %}
+{% block pageContent %}
+<div class="three half unit row">
+    <ul>
+        <li>
+            Hack Cambridge is an event running from 09.00 on 28<sup>th</sup> January 2017 - 17.00 on 29<sup>th</sup> January 2017 in Cambridge, UK. It will henceforth be referred to as 'the Event' or 'Hack Cambridge'.
+        </li>
+        <li>
+            The Organisers of Hack Cambridge, henceforth referred to as 'the Organisers' in this T&amp;C, refer to any or all of the individuals involved with organising the Event in any capacity.
```

Comment:
We should consider using smart quotes throughout this document

Reply:
Yes I wondered about that. The original was actually inconsistent. Presumably we should also make sure the document is up to date?

Reply:
If you'd rather, we could improve the text of the terms and conditions in another pull, since it's not really related to the ternary styling transition.  We can create issues for both those tasks

Reply:
Yes that makes sense

Reply:
#259